solr/core/src/test/org/apache/solr/cloud/overseer/ZkStateReaderTest.java

```diff
@@ -60,7 +62,7 @@ public void testExternalCollectionWatchedNotWatched() throws Exception{
 
       // create new collection
       ZkWriteCommand c1 = new ZkWriteCommand("c1",
-          new DocCollection("c1", new HashMap<>(), new HashMap<>(), DocRouter.DEFAULT, 0));
+          new DocCollection("c1", new HashMap<>(), Collections.singletonMap(ZkStateReader.CONFIGNAME_PROP, ConfigSetsHandler.DEFAULT_CONFIGSET_NAME), DocRouter.DEFAULT, 0));
```

Comment:
As we get used to Java 11, I think we'll prefer `Map.of` which is shorter.  This is still fine though.